flower_classifier/datasets/oxford_flowers.py

```diff
@@ -0,0 +1,93 @@
+import logging
+
+import torch
+from pathlib import Path
+
+from torch.utils.data import Dataset
+import torchvision
+from PIL import Image
+from scipy.io import loadmat
+
+logger = logging.getLogger(__name__)
```

Comment:
Technically, yes it still executes the logging statement. However, because we haven't set up a logging handler yet, you won't see any output. I'll go ahead and add the setup for basic logging in the root `__init__.py` so we can see the logging statements.